platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -198,6 +200,16 @@
 		.de("Wochentag des DBMS-Wartungsfensters");
 	I18n0 DEACTIVATED = new I18n0("deactivated")//
 		.de("deaktiviert");
+	I18n1 DELETES_AJAX_EXCEPTIONS_OLDER_THAN_ARG1_DAYS = new I18n1("Deletes AJAX Exceptions older than %s days")//
+		.de("Löscht AJAX-Ausnahmen, die älter als %s Tage sind");
+	I18n0 DELETES_EXECUTION_RECORDS_OF_ALL_PROGRAMS_ACCORDING_TO_THEIR_EXECUTION_RETENTION_DAYS_VALUE =
+			new I18n0("Deletes execution records of all programs according to their execution retention days value")//
+				.de("Löscht Ausführungsdatensätze aller Programme entsprechend dem Wert ihrer Speicherungstage für Ausführungen");
+	I18n1 DELETES_FILES_OLDER_THAN_ARG1_DAYS_FROM_FILE_SERVER_THAT_ARE_NO_LONGER_REFERENCED_IN_THE_DATABASE =
+			new I18n1("Deletes files older than %s days from file server that are no longer referenced in the database")//
```

Comment:
here also, should be plural `servers` and a period

Reply:
I changed it to plural but did not add a period because it's not a full sentence. I think, the description should be as short as possible.

Reply:
see above